src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+ul {
+  display: flex;
+  list-style: none;
+}
+
+.moyo_logo {
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  position: relative;
+  left: 0;
+}
+
+.nav__list {
+  text-align: center;
+  justify-content: center;
+}
+
+.nav__item {
+  margin-left: 20px;
+  display: flex;
+  font-family: "Roboto", sans-serif;
```

Comment:
you should set it for the entire page (html/body tag level) and explicitly inherit it if needed (if the auto inheritance is overwritten by default browser style)

also, it doesn't work at all now, because you haven't added this font-family to your page (use link to add it with google fonts)